src/components/ImagePreviewCard/ImagePreviewCard.tsx

```diff
@@ -0,0 +1,170 @@
+        as={chakra.button}
+        overflowY="hidden"
+        __css={styles.container}
+        p={0}
+        gridTemplateRows="1fr 4.5rem"
+        gridTemplateColumns="1fr 1fr"
+        gridTemplateAreas="'image image' 'footer footer'"
+        borderRadius="0.25rem"
+        _hover={{ bg: undefined }}
+        backgroundColor="base.canvas.overlay"
+        borderColor={isSelected ? "base.divider.brand" : undefined}
+        borderWidth={isSelected ? "2px" : undefined}
+      >
+        <GridItem gridArea="image">
+          <Box position="relative">
+            {/* White overlay over image */}
+            <Box
+              position="absolute"
+              h="100%"
+              w="100%"
+              left="0"
+              top="0"
+              pointerEvents="none"
+              backgroundColor="white"
+              opacity="0"
+              _groupHover={{ opacity: 0.25 }}
+            />
+
+            <Center>
+              <Image
+                align="center"
+                height="15rem"
+                src={mediaUrl}
+                fallbackSrc="/placeholder_no_image.png"
+                pointerEvents="all"
+                backgroundColor="white"
+              />
+            </Center>
+          </Box>
+        </GridItem>
+        <GridItem
+          gridArea="footer"
+          paddingInline="1rem"
+          py="0.875rem"
+          borderTopWidth="1px"
+          borderColor="border.divider.alt"
+          bgColor="base.canvas.default"
+          _groupHover={{
+            bgColor: "base.canvas.brand-subtle",
+          }}
+          w="100%"
+        >
+          <Grid
+            gridTemplateColumns={isMenuNeeded ? "1fr 2.75rem" : "1fr 0px"}
+            gridTemplateAreas="'content button'"
+          >
+            <VStack
+              alignItems="flex-start"
+              gridArea="content"
+              spacing="0.25rem"
+            >
+              <Text
+                textStyle="subhead-1"
+                textColor="base.content.strong"
+                textAlign="left"
+                noOfLines={1}
+              >
+                {name}
+              </Text>
+              <Text textStyle="caption-1" textColor="base.content.medium">
+                Added {relativeTime}
+              </Text>
+            </VStack>
+            <Box gridArea="button" />
+          </Grid>
+        </GridItem>
+      </Grid>
+      {isMenuNeeded && (
+        <ContextMenu>
+          <ContextMenu.Button />
+          <ContextMenu.List>
+            <ContextMenu.Item
+              icon={<BiEditAlt />}
+              // as={RouterLink}
+              // to={`${url}/editMediaSettings/${encodedName}`}
+            >
+              <Text>Rename image</Text>
+            </ContextMenu.Item>
+            <ContextMenu.Item
+              icon={<BiTrash />}
+              color="interaction.critical.default"
+              // as={RouterLink}
+              // to={`${url}/deleteMedia/${encodedName}`}
```

Comment:
maybe can delete

Reply:
Hmm I kept it there because this component is still work-in-progress and was intending to leave it as a placeholder to handover to @harishv7.